readme.md

```diff
@@ -18,15 +18,15 @@ Create a pages with product card using `flexbox`, `BEM` and `SCSS` based on [thi
 
 ## Checklist
 
-❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
+❗️ Replace `InnaPopenko` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_product-cards/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_product-cards/report/html_report/)
+- [DEMO LINK](https://InnaPopenko.github.io/layout_product-cards/)
+- [TEST REPORT LINK](https://InnaPopenko.github.io/layout_product-cards/report/html_report/)
```

Comment:
Add these links to the PR description
<img width="959" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/0e9a79af-b049-4654-80b1-1d0df4ce9085">

Reply:
everything is fixed